aptos-move/move-examples/fungible_asset/pre_minted_managed_coin/sources/preminted_managed_coin.move

```diff
@@ -0,0 +1,142 @@
+/// Same as `managed_coin` except that the total supply of coins are pre-minted when publishing the module. Nobody has
```

Comment:
Instead of saying "same as ...", I think it'd be better to list all the features that this supports here. It's more work but would be much less work for devs to understand what this does and whether it fits their needs